lib/ingestor-api/runtime/src/schemas.py

```diff
@@ -73,19 +73,6 @@ class Ingestion(BaseModel):
 
     item: Union[Item, Json[Item]]
 
-    class Config:
-        json_encoders = {
-            # Custom JSON serializer to ensure that item encodes as string.
-            # NOTE: when serializing, must call as ingestion.json(models_as_dict=False)
-            Item: lambda item: item.json(by_alias=True),
-        }
-
-    def json(self, *args, **kwargs):
-        # Update default to not represent models (e.g. `items` property) as a dict to
-        # allow our `json_encoders` override to properly serialize `items` property
-        kwargs.setdefault("models_as_dict", False)
-        return super().json(*args, **kwargs)
-
```

Comment:
@sharkinsspatial My rationale here is that if we need to put all this cruft in to make special considerations for serializing our `Ingestion` object for DynamoDB, it would probably make for cleaner code if we just put that cruft into the `dynamodb_dict()` method.